lib/worker.js

```diff
@@ -132,6 +132,15 @@ class AssetComputeWorker {
         if (!process.env.ASSET_COMPUTE_SDK_DISABLE_CGROUP_METRICS) {
             this.cgroupSampler = new Sampler(() => {
                 const metrics_object = cgroupMetrics();
+                const memoryUsage = metrics_object.memory.containerUsagePercentage;
```

Comment:
I think we should remove that before merging to master.
This was more a trial hack - we should come up with something cleaner and nicer later.